blocks/library/embed/index.js

```diff
@@ -216,14 +229,21 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 		},
 
 		save( { attributes } ) {
-			const { url, caption, align } = attributes;
+			const { url, caption, align, type, providerNameSlug } = attributes;
 
 			if ( ! url ) {
 				return;
 			}
 
+			const embedClassName = classnames(
+				'wp-block-embed',
+				align ? `align${ align }` : null,
```

Comment:
A nice feature of `classnames` is passing an object for optionally included class names:

```js
const embedClassName = classnames( 'wp-block-embed', {
	[ `align${ align }` ]: align,
	[ `type-${ type }` ]: type,
	[ `provider-${ providerNameSlug }` ]: providerNameSlug,
} );
```

https://www.npmjs.com/package/classnames

Reply:
Updated to reflect the optionally included class names syntax.